docs/docs/users/reference/cli.md

```
@@ -671,6 +673,41 @@ Options:
   -h, --help  Print help
 ```
 
+### `forest-cli mpool nonce-fix`
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)